### ISSUE TITLE
dress vol.1, save clothes soft locks the game

### ISSUE THREAD

**Issue by Triticum0** [RPCS3/rpcs3#14076]
### Quick summary

when saving clothes designs on the demo the game soft locks never return back house. 

### Details

https://github.com/RPCS3/rpcs3/assets/71127352/86ad9b70-2c7f-47bf-8e78-3486d01337f8

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/11847238/RPCS3.log.gz)

### System configuration

RPCS3 v0.0.28-15225-715e3856 Alpha | master | FW 4.88 | 5.15.106-1-MANJARO
AMD Ryzen 5 2600 | 12 Threads | 15.95 GiB RAM | AVX+ | FMA3
GPU: NVIDIA GeForce RTX 3060 Ti (530.41)

### Other details

Haven't tested the game on hardware so can't tell you if this is a bug or not

**Comment by FlexBy420** [comment]
Is this still an issue?

**Comment by Triticum0** [comment]
yes, still needs to be tested on hardware to confirm if this is a bug or not could be related to psn implementation.

**Comment by digant73** [comment]
is that still an issue?

**Comment by digant73** [comment]
@Triticum0 please provide an update on this issue
